src/util/Injected.js

```diff
@@ -80,6 +80,14 @@ exports.ExposeStore = (moduleRaidStr) => {
         };
     }
 
+    // VERSION >= 2.2330.X
```

Comment:
Can you use `window.WWebJS.compareWwebVersions` to check if the version allowing that feature?

Reply:
I am not 100% sure which version added that specific method. Besides, checking the method itself could be a safer play.